apps/omg_watcher/lib/omg_watcher/block_getter.ex

```diff
@@ -125,20 +125,19 @@ defmodule OMG.Watcher.BlockGetter do
         |> Core.ensure_block_imported_once(state)
         |> Enum.each(&DB.Transaction.update_with/1)
 
-      state = run_block_download_task(state)
-
-      {:ok, db_updates_from_state} = OMG.State.close_block(eth_height)
+      exit_processor_results = ExitProcessor.check_validity()
```

Comment:
Requesting @pthomalla review for just only this file - remember recently he fixes some nasty things around block getter...

Reply:
@pthomalla please take a look

Reply:
ExitProcessor.check_validity () should be on the end,
otherwise, users will learn about invalid block (invalid exit)
at the next block. So the users can find out the possibility of challenge
later than they should

Reply:
This implies testing as below average because if your requested changes make sense-at least one test would need to fail, but it didn’t. 

And if we want to be comfortable with making changes, we need to be sure they cover the behavior. 

So let us formulate a test that covers the behavior you’re implying needs to stay intact. How would the test look like @pthomalla ?